docs/how-tos/how-to-use-a-custom-promotion-adjuster.mdx

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 7
+---
+
+# How to use a custom promotion adjuster
+
+Every time an order is updated, its eligible promotions might
+not be valid anymore; for example, this could happen if a promotion is
+tight to a specific order total threshold, and this value changes within
+the update. In the same way, changing the order could make it eligible
+for new promotions.
+
+In this guide, we'll see how to create a custom promotion adjuster to
```

Comment:
What do you think about moving this paragraph to the top, so it's the first one?